distributions/openhab/pom.xml

```diff
@@ -170,6 +170,7 @@
                         <feature>system</feature>
                         <feature>openhab-runtime-base</feature>
                         <feature>openhab-runtime-ui</feature>
+                        <feature>openhab-addons-discovery</feature>
```

Comment:
Should this be `openhab-addon-discovery` without "s" ??

Reply:
I decided to use `addon`. But it doesn't matter as long as they are all the same.